<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_11">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_12">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_13">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_14">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_15">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_16">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_17">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_18">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_19">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_20">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_21">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_22">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_23">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_24">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_25">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_26">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_27">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_28">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_29">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_30">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_31">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_32">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_33">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_34">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_35">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_36">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_37">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_38">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_39">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_40">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_41">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_42">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_43">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_44">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_45">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_46">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_47">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_48">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_49">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_50">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_51">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_52">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_53">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_54">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_55">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_56">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_57">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_58">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_59">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_60">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_61">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_62">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_63">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_64">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_65">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_66">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_67">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_68">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_69">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_70">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_71">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_72">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_73">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_74">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_75">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_76">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_77">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_78">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_79">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_80">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>270</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_81">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="solve">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>solve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
